guides/common/modules/proc_resetting-custom-ssl-certificate-to-default-self-signed-certificate-on-project.adoc

```diff
@@ -11,10 +11,10 @@
 
 .Verification
 ifndef::orcharhino[]
-Ensure the following parameters in `/etc/foreman-installer/scenarios.d/{project-context}-answers.yaml` have no values:
```

Comment:
I checked `--full-help` and didn't find "current" values for these parameters. Only the default ones but that's not what we want users to check here. However, I believe "Ensure" is not the right action verb to use here; rather than encourage users to delete the values in the file, we want them to make sure that no values are currently set.